cloudregister/defaults.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2025, SUSE LLC, All rights reserved.
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3.0 of the License, or (at your option) any later version.
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library.
+#
+#
+LOG_FILE = '/var/log/cloudregister'
+ZYPP_SERVICES = '/etc/zypp/services.d'
```

Comment:
If we are skinning the cat this way then we should probably pull all the constants that are in the utils file into this file.

Reply:
yes I agree and this is also my plan for further PRs. The reason why I did it only partially was because this PR was already super hard to review and as such not everything was moved.